js/components/communityresource/list.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+    <datatable title="{{ title }}" icon="code-fork"
+        boxclass="community-widget"
+        fields="{{ fields }}"
+        p="{{ communities }}"
+        empty="{{ _('No community resources') }}">
+    </datatable>
+</template>
+
+
+<script>
+import Vue from 'vue';
+import CommunityResource from 'models/communityresource';
+
+export default {
+    name: 'community-widget',
+    props: ['communities', 'withoutDataset'],
```

Comment:
WHy not using resource(s) everywhere instead of communit(y|ies) ? It seem a little weird to me :/